contact.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Mafalda Portfolio</title>
+
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/css/style.css">
+</head>
+
+<body>
+
+
+    <div class="menu">
+        <a class="menu_top_left" href="index.html" > About </a>
```

Comment:
Instead of linking straight to the `.html` file, remember to use relative pathing to route to other pages! `href="/portfolio"` will work too, you can also `href="/"` to go back to the top page

Reply:
Oh! I see thanks for the feedback!